app/webpacker/components/CompetitionsOverview/CompetitionsFilters.js

```diff
@@ -52,12 +54,22 @@ function CompetitionsFilters({
         </Form.Field>
       </Form.Group>
 
+      {canViewAdminDetails && shouldShowAdminDetails && (
+        <Form.Group>
+          <Form.Field>
+            <AdminStatusButtonGroup filterState={filterState} dispatchFilter={dispatchFilter} />
+          </Form.Field>
+        </Form.Group>
+      )}
```

Comment:
Can we put this below the checkboxes? It feels weird when clicking a checkbox moves that checkbox down, so you can't directly click again to un-check it.

(Just my initial thoughts from running locally; will review the code later.)